ricoconverter/ricoconverter-convert/src/main/resources/xslt_eac/eac2rico.xslt

```diff
@@ -654,6 +659,7 @@
 				<xsl:apply-templates />
 			</rico:PerformanceRelation>
 		</rico:agentIsTargetOfPerformanceRelation>
```

Comment:
I suggest we add the following line here:
`<rico:hasOrHadOccupationOfType>
   <xsl:call-template name="rdf-resource"><xsl:with-param name="uri" select="eac2rico:URI-OccupationType(eac:term/@vocabularySource)" /></xsl:call-template>
</rico:hasOrHadOccupationOfType>`